lib/kubeclient/informer.rb

```diff
@@ -70,15 +71,17 @@ def cache_key(resource)
     end
 
     def fill_cache
-      reply = @client.get_entities(nil, @resource_name, raw: true, resource_version: '0')
+      get_options = @options.merge(raw: true, resource_version: '0')
```

Comment:
@grosser similar question on the origin of `raw: true` here, I don't see it doing anything.
Did you intend `as: :raw`?  Though that means totally unparsed string, I suspect you want `as: :parsed_symbolized` and the current code works thanks to default RecursiveOpenStruct's leniency (?)